examples/demo_all.pp

```diff
@@ -31,6 +31,7 @@
   include ciscopuppet::demo_command_config
   include ciscopuppet::demo_domain
   include ciscopuppet::demo_interface
+  #include ciscopuppet::demo_interface_service
```

Comment:
Too many prerequisites to use it on most platforms right now. It needs support for VDCs and special linecards.